🌐 Update translations for uk (update-outdated) (#14822)
fastapi/fastapi: docs/uk/docs/index.md

<a href="https://typer.tiangolo.com" target="_blank"><img src="https://typer.tiangolo.com/img/logo-margin/logo-margin-vector.svg" style="width: 20%;"></a>

> Якщо ви створюєте застосунок <abbr title="Command Line Interface - Command Line Interface">CLI</abbr> для використання в терміналі замість веб-API, зверніть увагу на <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>.

Reviewer: ```suggestion
Якщо ви створюєте застосунок <abbr title="Command Line Interface">CLI</abbr> для використання в терміналі замість веб-API, зверніть увагу на <a href="https://typer.tiangolo.com/" class="external-link" target="_blank">**Typer**</a>.
```

* Валідацію даних:
    * Автоматичні та зрозумілі помилки, коли дані некоректні.
    * Валідацію навіть для глибоко вкладених JSON-обʼєктів.

> * <abbr title="also known as: serialization, parsing, marshalling - також відомо як serialization, parsing, marshalling">Перетворення</abbr> вхідних даних: з мережі до даних і типів Python. Читання з:

Reviewer 2: It may be in this way: `<abbr title="також відоме як serialization, parsing, marshalling">`. Also, it can be `<abbr title="відоме як serialization, parsing, marshalling">`, but we're losing a word "also".

Also, there is a same problem on line 43. I explained reasons of the problem there.

UPD: added a variant and explained why is it correct too

> * <abbr title="also known as: serialization, parsing, marshalling - також відомо як serialization, parsing, marshalling">Перетворення</abbr> вхідних даних: з мережі до даних і типів Python. Читання з:

Reviewer: ```suggestion
* <abbr title="також відоме як serialization, parsing, marshalling">Перетворення</abbr> вхідних даних: з мережі до даних і типів Python. Читання з:
```

    * JSON.
    * Параметрів шляху.
    * Параметрів запиту.
    * Cookies.
    * Headers.
    * Forms.
    * Files.

> * <abbr title="also known as: serialization, parsing, marshalling - також відомо як serialization, parsing, marshalling">Перетворення</abbr> вихідних даних: перетворення з даних і типів Python у мережеві дані (як JSON):

Reviewer: Same problem with `<abbr>` as above

> * <abbr title="also known as: serialization, parsing, marshalling - також відомо як serialization, parsing, marshalling">Перетворення</abbr> вихідних даних: перетворення з даних і типів Python у мережеві дані (як JSON):

Reviewer: ```suggestion
* <abbr title="також відоме як serialization, parsing, marshalling">Перетворення</abbr> вихідних даних: перетворення з даних і типів Python у мережеві дані (як JSON):
```

Він забезпечує той самий **developer experience** створення застосунків на FastAPI під час їх **розгортання** у хмарі. 🎉

FastAPI Cloud - основний спонсор і джерело фінансування open source проєктів *FastAPI and friends*. ✨

#### Розгортання в інших хмарних провайдерів { #deploy-to-other-cloud-providers }

> FastAPI - open source і базується на стандартах. Ви можете розгортати застосунки FastAPI в будь-якому хмарному провайдері, який ви оберете.

Reviewer: I personally think it's Ok to translate it as "open source проект" ("open source project"). I think it sounds clearer than "відкритий код" (more like "open source code").
I think we should go with "open source проект"

> FastAPI - open source і базується на стандартах. Ви можете розгортати застосунки FastAPI в будь-якому хмарному провайдері, який ви оберете.

Reviewer 2: OK, no problem! We can leave "open source проект"

* <a href="https://www.python-httpx.org" target="_blank"><code>httpx</code></a> - потрібно, якщо ви хочете використовувати `TestClient`.
* <a href="https://jinja.palletsprojects.com" target="_blank"><code>jinja2</code></a> - потрібно, якщо ви хочете використовувати конфігурацію шаблонів за замовчуванням.

> * <a href="https://github.com/Kludex/python-multipart" target="_blank"><code>python-multipart</code></a> - потрібно, якщо ви хочете підтримувати <abbr title="converting the string that comes from an HTTP request into Python data - перетворення строки, що надходить з HTTP-запиту, у дані Python">«parsing»</abbr> форм за допомогою `request.form()`.

Reviewer 3: ```suggestion
* <a href="https://github.com/Kludex/python-multipart" target="_blank"><code>python-multipart</code></a> - потрібно, якщо ви хочете підтримувати <abbr title="converting the string that comes from an HTTP request into Python datatypes - перетворення строки із HTTP-запиту, у Python типи">«parsing»</abbr> форм за допомогою `request.form()`.
```

> * <a href="https://github.com/Kludex/python-multipart" target="_blank"><code>python-multipart</code></a> - потрібно, якщо ви хочете підтримувати <abbr title="converting the string that comes from an HTTP request into Python data - перетворення строки, що надходить з HTTP-запиту, у дані Python">«parsing»</abbr> форм за допомогою `request.form()`.

Reviewer: ```suggestion
* <a href="https://github.com/Kludex/python-multipart" target="_blank"><code>python-multipart</code></a> - потрібно, якщо ви хочете підтримувати <abbr title="перетворення строки із HTTP-запиту, у Python типи">«parsing»</abbr> форм за допомогою `request.form()`.
```

It should be just translation, without English version
